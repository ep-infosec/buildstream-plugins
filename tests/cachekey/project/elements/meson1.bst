kind: meson
sources:
- kind: local
  path: files/local
