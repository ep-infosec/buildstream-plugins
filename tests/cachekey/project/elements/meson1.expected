e971e99926950e605d88f3d65edd1bad584a2b2a23943460aa48250be889622f